pkg/controller/apiserver/handlers/handler_test.go

```diff
@@ -195,8 +194,21 @@ func evaluateTestCases(testCases map[string]TestCase, mockCtrl *gomock.Controlle
 	}
 }
 
-func genHTTPRequest(args []string) string {
-	return "?name=" + args[1] + "&&namespace=" + args[0]
+func genHTTPRequest(args []string) (request string) {
```

Comment:
have you considered using the actual request args as the `handlerArgs` in your test cases, e.g. `?namespace=<namespace>?pod=<pod>?` instead of `[]string{<namespace>, <pod>}`?

I am not sure what the other tests are doing, but it may make the tests less verbose and easier to read.

Let me know what you think.

Reply:
I have made the changes.